src/Serilog.Sinks.Fluentd/Serilog.Sinks.Fluentd.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
```

Comment:
dont change standard version. You force other consumers with obsolete versions to upgrade their apps

Reply:
reverted